.github/scripts/check-format.sh

```diff
@@ -42,7 +42,8 @@ else
 fi
 
 find . -type d \( \
-    -path ./\*build -o \
+    -path ./\*build\* -o \
+    -path ./release \
```

Comment:
Shouldn't there be another -o after release?

Reply:
Fixed